README.md

```diff
@@ -3,18 +3,18 @@
+The main focus of the package is to transform the core object tables into analytics-ready models. It includes options to explore video demographics and a comprehensive overview of video performance that you could combine with other organic ad platform reports.
 
 ## Models
 This package contains transformation models, designed to work simultaneously with our [YouTube Analytics source package](https://github.com/fivetran/dbt_youtube_analytics_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
 
 | **model**                 | **description**                                                                                                    |
 | ------------------------- | ------------------------------------------------------------------------------------------------------------------ |
-| [youtube__video_report](models/youtube__video_report.sql)  | Each record represents daily aggregation of your YouTube video performance.    |
-| [youtube__demographics_report](models/youtube__demographics_report.sql)        | Each record represents a daily capture of video view percentage across the gender, age, and country demographics.            |
+| [youtube__video_report](models/youtube__video_report.sql)  | Each record represents the daily aggregation of your YouTube video performance.    |
+| [youtube__demographics_report](models/youtube__demographics_report.sql)        | Each record represents a daily video view percentage by gender, age, and country.            |
 | [youtube__age_demographics_pivot](models/youtube__age_demographics_pivot.sql)        | Each record represents a daily video view percentage with the age ranges pivoted out for quicker analysis.            |
 | [youtube__gender_demographics_pivot](models/youtube__gender_demographics_pivot.sql)        | Each record represents a daily video view percentage with the gender options pivoted out for quicker analysis.            |
-| [youtube__video_metadata](models/youtube__gender_demographics_pivot.sql)           | Each record represents an individual video enriched with metadata. This model is disabled by default. Refer to our [YouTube Video Metadata Cloud Function Write Up](write-up_to_go_here) or our [YouTube Analytics source package](https://github.com/fivetran/dbt_youtube_analytics_source) for more details on how to generate this model.          |
+| [youtube__video_metadata](models/youtube__gender_demographics_pivot.sql)           | Each record represents an individual video enriched with metadata. This model is disabled by default. Read our [YouTube Video Metadata Cloud Function Write Up](write-up_to_go_here) or our [YouTube Analytics source package README](https://github.com/fivetran/dbt_youtube_analytics_source) to learn how to generate this model.          |
```

Comment:
Please update this link when the one-pager is ready

Reply:
Will do, thanks!